src/qminer/qminer_aggr.h

```diff
@@ -871,6 +904,46 @@ class TChiSquare : public TStreamAggr, public TStreamAggrOut::IFlt {
 	TStr Type() const { return GetType(); }
 };
 
+///////////////////////////////
+/// TMinCountSketch stream aggregate.
+class TCountMinSketch : public TStreamAggr, public TStreamAggrOut::IFlt {
+private:
+	// input
+	TWPt<TStreamAggr> InAggr;
+	TWPt<TStreamAggrOut::IFltTm> InAggrVal;
+	// indicator
+	TSignalProc::TCountMinSketch Model;
+	TIntV Vals;
+protected:
+	void OnAddRec(const TRec& Rec);
+	TCountMinSketch(const TWPt<TBase>& Base, const PJsonVal& ParamVal);
+public:
+	static PStreamAggr New(const TWPt<TBase>& Base, const PJsonVal& ParamVal);
+	// did we finish initialization
+	bool IsInit() const { return InAggr->IsInit(); }
+	/// Reset
+	void Reset() {  }
+	/// Load from stream
+	void LoadState(TSIn& SIn);
+	/// Store state into stream
+	void SaveState(TSOut& SOut) const;
+	/// get current count value
+	double GetFlt() const {	return Model.Estimate(Vals[0]); }
+	/// get count value
+	double GetFlt(const TInt& Val) const {	return Model.Estimate(Val); }
```

Comment:
Remove GetFlt, GetFlt(...).
Add GetIntFun interface to qminer_core.h

class IIntFunc {
	public:
		// functions that map ints to ints
		virtual int GetIntFunc(const int& Arg) const = 0;
	};